skxray/diff_roi_choice.py

```diff
@@ -319,28 +325,55 @@ def _grid_values(img_dim, calibrated_center):
     return grid_values
 
 
-def _process_rings(num_rings, img_dim, ring_vals, ring_inds):
+def _process_rings(num_rings, ring_vals):
     """
-    This will find the indices of the required rings, find the bin
-    edges of the rings, and count the number of pixels in each ring,
-    and pixels list for the required rings.
+    This is a helper function to make edge values
+     of the each roi ring shape as (num_rings, 2)
 
     Parameters
     ----------
     num_rings : int
         number of rings
 
-    img_dim: tuple
-        shape of the image (detector X and Y direction)
-        Order is (num_rows, num_columns)
-        shape is [detector_size[0], detector_size[1]])
+    ring_vals : ndarray
+        edge values of each ring
 
+    Returns
+    -------
     ring_vals : ndarray
         edge values of each ring
+        shape is (num_rings, 2)
 
-    ring_inds : ndarray
-        indices of the required rings
-        shape is ([detector_size[0]*detector_size[1]], )
+    """
+    ring_vals = np.asarray(ring_vals).reshape(-1, 2)
+
+    return ring_vals
+
+
+def roi_divide_circle(detector_size, radius, calibrated_center,
+                      num_angles, rotate='N'):
+    """
+    This module will provide the indices, number of pixels and
+    pixel indices when a circular roi is divided into pies
+    Parameters
+    ----------
+    detector_size : tuple
+        shape of the image (detector X and Y direction)
+        Order is (num_rows, num_columns)
+
+    radius : float
+        radius of the circle
+
+    calibrated_center : tuple
+        defining the center of the image
+        (column value, row value) (mm)
+
+    num_angles : int
+        number of angles ring divide into
+        angles are measured from horizontal-anti clock wise
+
+    rotate : {'Y', 'N'}, optional
```

Comment:
If we keep angle as a user input I don't think it will be a good idea. Because I want to divide the circle into same size wedges and later use that different wedges roi information(pixel indices, indices..) to run one time correlation and see whether there is a difference between those wedges correlation plots.

I kept the rotation as a optional input